### PR TITLE
Specify AMI root volume device name for all distributions

### PR DESCRIPTION
**Pull request by randomvariable**
Amazon Linux 2 images created after #293 were non bootable on non-nvme instance types. This is because `/dev/sda1` ended up being a blank volume overriding the `/dev/xvda` root volume. This PR creates a `root_device_name` variable, which is then set for each distribution.

### CODE REVIEW

File: images/capi/packer/ami/packer.json [hunk 1]
```diff
@@ -141,13 +141,15 @@
       "format_options": "{{user `goss_format_options`}}",
       "goss_file": "{{user `goss_entry_file`}}",
       "inspect": "{{user `goss_inspect_mode`}}",
-      "tests": ["{{user `goss_tests_dir`}}"],
+      "tests": [
```
Comment (dims) [inline images/capi/packer/ami/packer.json:144]: unrelated whitespace change?

File: images/capi/packer/ami/packer.json [hunk 2]
```diff
@@ -141,13 +141,15 @@
...
+        "{{user `goss_tests_dir`}}"
+      ],
       "url": "{{user `goss_url`}}",
       "use_sudo": true,
       "vars_inline": {
         "kubernetes_source_type": "{{user `kubernetes_source_type`}}",
         "kubernetes_cni_source_type": "{{user `kubernetes_cni_source_type`}}",
-        "containerd_version" : "{{user `containerd_version`}}",
+        "containerd_version": "{{user `containerd_version`}}",
```
Comment (dims) [inline images/capi/packer/ami/packer.json:152]: same as above?

File: images/capi/packer/ami/packer.json (hunk 1 above)
Comment (randomvariable) [reply images/capi/packer/ami/packer.json:144]: it got linted automatically, have been done as two commits.

Comment (dims) [reply images/capi/packer/ami/packer.json:144]: ah missed that :) thanks

File: images/capi/packer/ami/packer.json (hunk 2 above)
Comment (randomvariable) [reply images/capi/packer/ami/packer.json:152]: same as above. i can delete the linting commit if needed.
